server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -1781,7 +1781,7 @@ public void run() {
                         final var nodeEligibility = localNodeMayWinElection(lastAcceptedState, electionStrategy);
                         if (nodeEligibility.mayWin() == false) {
                             assert nodeEligibility.reason().isEmpty() == false;
-                            logger.trace(
+                            logger.info(
                                 "skip prevoting as local node may not win election ({}): {}",
```

Comment:
You'll need to follow up with a serverless test fix to testShutdownLogMessageDuringElectionAttempt, which expects a TRACE log (also activates TRACE logging via @TestLogging for that check)

Reply:
Ha, I just saw your PR for this. Nvm 🙂